src/main/java/mate/academy/library/LibraryApplication.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.library;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class LibraryApplication {
+    public static void main(String[] args) {
+        SpringApplication.run(LibraryApplication.class, args);
+    }
+
```

Comment:
```suggestion
```
be careful with extra empty lines, review each file that is meant to be committed